### PR TITLE
PDF progress up until now, stable

### PR DESCRIPTION
**Pull request by DarjushKolahi**
No charts are generated and richtextbox is not implemented yet, but the rest of the PDF functionality works correctly as far as I'm concerned.

### CODE REVIEW

File: FSBeheer/FSBeheer/App.config [hunk 1]
```diff
@@ -26,7 +26,7 @@
     </providers>
   </entityFramework>
   <connectionStrings>
-    <!--<add name="FSDB" connectionString="data source=localhost;initial catalog=festispec;integrated security=true;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />-->
-    <add name="FSDB" connectionString="data source=tcp:festispec2.database.windows.net,1433;initial catalog=festispec;user id=berm;password=Bart_2018!;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />
+    <add name="FSDB" connectionString="data source=localhost;initial catalog=festispec;integrated security=true;MultipleActiveResultSets=True;App=EntityFramework" providerName="System.Data.SqlClient" />
```
Comment (Evert-Arends) [inline FSBeheer/FSBeheer/App.config]: Heeeeelp.... I neeeeeeed somee booodddyyyy

**Review by Evert-Arends — APPROVED** [review]

File: FSBeheer/FSBeheer/App.config (hunk 1 above)
Comment (DarjushKolahi) [reply FSBeheer/FSBeheer/App.config]: My bad, veranderd

Comment (Evert-Arends) [reply FSBeheer/FSBeheer/App.config]: Dit

**Review by Evert-Arends — CHANGES_REQUESTED** [review]

**Comment by DarjushKolahi** [comment]
ik heb de connectionstring al veranderd als het goed is

**Comment by DarjushKolahi** [comment]
zie commit a9eeb3d

**Pull request merged**